bifunctors.cabal

```diff
@@ -54,42 +39,27 @@ flag tagged
 
 library
   hs-source-dirs: src
-  include-dirs: include
-  includes: bifunctors-common.h
   build-depends:
-    base                >= 4.3   && < 5,
+    base                >= 4.9   && < 5,
     base-orphans        >= 0.8.4 && < 1,
     comonad             >= 5.0.7 && < 6,
     containers          >= 0.2   && < 0.7,
-    template-haskell    >= 2.4   && < 2.20,
+    template-haskell    >= 2.11  && < 2.20,
     th-abstraction      >= 0.4.2.0 && < 0.5,
-    transformers        >= 0.3   && < 0.7
+    transformers        >= 0.5   && < 0.7
 
   if !impl(ghc > 8.2)
     build-depends: transformers-compat >= 0.5 && < 0.8
```

Comment:
This is there to provide orphan instances through `Data.Bifoldable` and `Data.Bitraversable`.

However looking at https://hackage.haskell.org/package/transformers-compat-0.7.2/docs/src/Control.Monad.Trans.Instances.html and https://hackage.haskell.org/package/transformers-compat-0.7.2/changelog it looks the lower bound needs to be 0.6?

> Backport changes from transformers-0.5.4 (i.e., add Bifoldable and Bitraversable instances for Data.Functor.Constant)

other instances in `Control.Monad.Trans.Instances` don't seem to be relevant to `Bifunctor` family of classes

Reply:
I've bumped the lower version bounds on `transformers-compat` to `0.6`.